source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -180,81 +170,71 @@ Note:
 ## Exporting a VM or Template as an OVA
 The user will be able to select a VM (or a template) to be exported as
 an OVA file. In such cases, the user needs to specify a path on a host
-where the OVA file will be created.
+or a disk on shared storage where the OVA file will be created.
 
 The following figure depicts the process of exporting a VM with a single
 disk as an OVA file:
 
 ![](../../../../images/wiki/export-ova.png)
 
-1. The front end sends the back end a VM and a path on a host.
-2. The back end asks imagio-proxy to initialize a file at the given path
-   with an estimated size of the produced OVA file.
-3. Imageio creates the OVA file.
-4. The disk is streamed from the back end to imageio-proxy, and then to
-   imageio.
-5. Imageio reads the disk and streams it into a QCOW volume within the
-   OVA.
-6. The complete OVF specification is sent from the back end to
-   imageio-proxy. imageio-proxy sends the specification to imageio, and
-   imageio streams it into the OVA file.
-
+1. The frontend sends the backend a VM and an export destination.
+2. Backend creates new disk (for the ova) if the destination requires.
+   The backend should also prepare and teardown the disks if needed.
+3. Backend asks VDSM to create an OVA file with the complete OVF 
+   specification and export destination.
+4. VDSM runs ovirt-ova tool - creates the OVA in the destination
+   (using qemu-img convert which may include file format conversion).
+   
 Notes:
 
-1. Step 5 is equivalent to `qemu-img convert` with the collapse option
+1. Step 4 is equivalent to `qemu-img convert` with the collapse option
    set and the target format set to QCOW.
-2. The OVF configuration needs to be streamed to the beginning of the
-   OVA file (according to the OVA specification, the OVF should be the
-   first tar entry inside the OVA file).
-3. The back end should be provided with the overall amount of data
-   written to the disk entry in step 4, because this information needs
-   to be set inside the OVF configuration.
-4. The OVA file will be created with a special extension (step 3). At
-   the end of the process (step 6), the file will be renamed, and the
-   extension dropped. That is done in order to enable tools that
-   synchronize shares between various deployments to differentiate
-   partial and complete files.
+2. Only after building the OVA file we could update the ovf with the
+   export size.
+3. If possible, we'll consider using streaming the disk data directly
+   from the qemu chain into OVA.
```

Comment:
This has major implication about the usability of the feature, I'm afraid.
It should be noted not as a note in this section, so it will be probably exposed. Remember that this wiki page was reviewed by PMs and other stackholders and this changes a major assumption of what it said before. I suggest to start exploring the streaming option, and if we see that it is not possible, then letting everyone know that we take this shortcut.